server/lib/nl/scraper.py

```diff
@@ -80,7 +84,7 @@
stroke%3A%20rgb%28255%2C%20255%2C%20255%29%3B%22%3E%3C/circle%3E%3Ccircle%20class%3D%22dot%22%20cx%3D%22375.8805855524371%22%20cy%3D%2245.35657142857143%22%20r%3D%223%22%20style%3D%22fill%3A%20rgb%28147%2C%200%2C%200%29%3B%20stroke%3A%20rgb%28255%2C%20255%2C%20255%29%3B%22%3E%3C/circle%3E%3Ccircle%20class%3D%22dot%22%20cx%3D%22294.39788985487405%22%20cy%3D%2245.479142857142854%22%20r%3D%223%22%20style%3D%22fill%3A%20rgb%28147%2C%200%2C%200%29%3B%20stroke%3A%20rgb%28255%2C%20255%2C%20255%29%3B%22%3E%3C/circle%3E%3Ccircle%20class%3D%22dot%22%20cx%3D%22212.69195389512598%22%20cy%3D%2244.737428571428566%22%20r%3D%223%22%20style%3D%22fill%3A%20rgb%28147%2C%200%2C%200%29%3B%20stroke%3A%20rgb%28255%2C%20255%2C%20255%29%3B%22%3E%3C/circle%3E%3Ccircle%20class%3D%22dot%22%20cx%3D%22131.20925819756297%22%20cy%3D%2244.027142857142856%22%20r%3D%223%22%20style%3D%22fill%3A%20rgb%28147%2C%200%2C%200%29%3B%20stroke%3A%20rgb%28255%2C%20255%2C%20255%29%3B%22%3E%3C/circle%3E%3C/g%3E%3Cpath%20class%3D%22line%22%20d%3D%22M538.845976947563%2C30.25828571428571L457.36328125%2C31.641142857142853L375.8805855524371%2C31.487142857142857L294.39788985487405%2C31.905142857142863L212.69195389512598%2C31.716571428571427L131.20925819756297%2C30.978L49.7265625%2C31.801428571428573%22%20style%3D%22stroke%3A%20rgb%2894%2C%2079%2C%20162%29%3B%20fill%3A%20none%3B%20stroke-width%3A%202.5px%3B%22%3E%3C/path%3E%3Cg%3E%3Ccircle%20class%3D%22dot%22%20cx%3D%22538.845976947563%22%20cy%3D%2230.25828571428571%22%20r%3D%223%22%20style%3D%22fill%3A%20rgb%2894%2C%2079%2C%20162%29%3B%20stroke%3A%20rgb%28255%2C%20255%2C%20255%29%3B%22%3E%3C/circle%3E%3Ccircle%20class%3D%22dot%22%20cx%3D%22457.36328125%22%20cy%3D%2231.641142857142853%22%20r%3D%223%22%20style%3D%22fill%3A%20rgb%2894%2C%2079%2C%20162%29%3B%20stroke%3A%20rgb%28255%2C%20255%2C%20255%29%3B%22%3E%3C/circle%3E%3Ccircle%20class%3D%22dot%22%20cx%3D%22375.8805855524371%22%20cy%3D%2231.487142857142857%22%20r%3D%223%22%20style%3D%22fill%3A%20rgb%2894%2C%2079%2C%20162%29%3B%20stroke%3A%20rgb%28255%2C%20255%2C%20255%29%3B%22%3E%3C/circle%3E%3Ccircle%20class%3D%22dot%22%20cx%3D%22294.39788985487405%22%20cy%3D%2231.905142857142863%22%20r%3D%223%22%20style%3D%22fill%3A%20rgb%2894%2C%2079%2C%20162%29%3B%20stroke%3A%20rgb%28255%2C%20255%2C%20255%29%3B%22%3E%3C/circle%3E%3Ccircle%20class%3D%22dot%22%20cx%3D%22212.69195389512598%22%20cy%3D%2231.716571428571427%22%20r%3D%223%22%20style%3D%22fill%3A%20rgb%2894%2C%2079%2C%20162%29%3B%20stroke%3A%20rgb%28255%2C%20255%2C%20255%29%3B%22%3E%3C/circle%3E%3Ccircle%20class%3D%22dot%22%20cx%3D%22131.20925819756297%22%20cy%3D%2230.978%22%20r%3D%223%22%20style%3D%22fill%3A%20rgb%2894%2C%2079%2C%20162%29%3B%20stroke%3A%20rgb%28255%2C%20255%2C%20255%29%3B%22%3E%3C/circle%3E%3Ccircle%20class%3D%22dot%22%20cx%3D%2249.7265625%22%20cy%3D%2231.801428571428573%22%20r%3D%223%22%20style%3D%22fill%3A%20rgb%2894%2C%2079%2C%20162%29%3B%20stroke%3A%20rgb%28255%2C%20255%2C%20255%29%3B%22%3E%3C/circle%3E%3C/g%3E%3C/g%3E%3C/svg%3E",
```

Comment:
what are these large blobs used for?

Reply:
This is the demo query used by bard that's "cached" without any backend processing.

Added a TODO to remove